### PR TITLE
Contfrac

### PR DESCRIPTION
**Pull request by jtlap**
this pr ads 2 functions able to evaluate continuous fractions which are needed by some implementation of special function (in particular in kyosu)

### CODE REVIEW

File: include/eve/module/contfrac/impl/lentz.hpp [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+namespace eve::detail
+{
+
+  template <typename T>
+  struct is_pair : public std::false_type{};
```
Comment (jfalcou) [inline include/eve/module/contfrac/impl/lentz.hpp:14]: you don't need public

File: include/eve/module/contfrac/impl/lentz.hpp [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+  template <typename T, typename U>
+  struct is_pair<kumi::tuple<T,U>> : public std::true_type{};
```
Comment (jfalcou) [inline include/eve/module/contfrac/impl/lentz.hpp:17]: same here, public is unneeded

**Review by jfalcou — CHANGES_REQUESTED** [review]
Can't this just be in math or does it depends of something else ?

File: include/eve/module/contfrac/impl/lentz.hpp (hunk 1 above)
Comment (jtlap) [reply include/eve/module/contfrac/impl/lentz.hpp:14]: ok

**Review by jfalcou — APPROVED** [review]

**Pull request merged**